src/main/java/robotbuilder/ParameterSetsTable.java

```diff
@@ -303,7 +303,7 @@ private boolean areParametersValid(int rowNum) {
         }
 
         @Override
-        public Vector<Vector<Object>> getDataVector() {
+        public Vector<Vector> getDataVector() {
```

Comment:
You shouldn't drop the generic on the second vector, just return `Vector<Vector<*>>` or `Vector<Vector<? extends Object>>` might work as well.

Reply:
I get a type not compatible error on Java 11 adding anything to the inner vector parameter.

Reply:
@JLLeitschuh Java 9 changed the signature of the base method from returning a raw vector to a typed vector. See #107 for details